### PR TITLE
New version

### PR DESCRIPTION
**Pull request by MebiusCat**

### CODE REVIEW

File: manager.py [hunk 1]
```diff
@@ -1,136 +1,140 @@
...
+        return 'Specify the file or directory'
+    name = args[0]
+    try:
+        if os.path.isfile(name):
+            os.remove(name)
         else:
-            # print(_user_path + '/' + _argument)
-            # print(_user_path, _argument)
-            print('Invalid command')
-            # Apparently I can push to main!
```
Comment (hypersunk) [inline manager.py:21]: I'm going to miss this line very much 💔

File: manager.py [hunk 2]
```diff
@@ -1,136 +1,140 @@
...
+    """
+    Copying files or directories
+    """
+
+    if len(args) > 2:
+        return 'Specify the current name of the file or directory and the new location and/or name'
+    elif len(args) != 2:
+        return 'Specify the file'
+    fr, to = args
+    if not os.path.isdir(fr) and not os.path.isfile(fr):
```
Comment (hypersunk) [inline manager.py:119]: this looks a bit suspicious but OKAY

**Review by hypersunk — CHANGES_REQUESTED** [review]

File: manager.py (hunk 1 above)
Comment (MebiusCat) [reply manager.py:21]: Me too

**Review by hypersunk — APPROVED** [review]
lgtm

**Pull request merged**